src/Apps/W1/Shopify/App/src/Order handling/Codeunits/ShpfyCopySalesDocument.Codeunit.al

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Integration.Shopify;
+
+using Microsoft.Sales.Document;
+using Microsoft.Utilities;
+
+codeunit 30402 "Shpfy Copy Sales Document"
+{
+    Access = Internal;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Copy Document Mgt.", 'OnAfterCopySalesHeaderArchive', '', false, false)]
+    local procedure OnAfterCopySalesHeaderArchive(var ToSalesHeader: Record "Sales Header")
+    begin
+        if (ToSalesHeader."Shpfy Order Id" = 0) and (ToSalesHeader."Shpfy Order No." = '') then
```

Comment:
What's the difference between exiting when they're 0 and empty, vs clearing? 
Is this just to avoid the modify if they're already cleared?